pipelines/strip.yaml

```diff
@@ -0,0 +1,34 @@
+name: Strip binaries
+
+needs:
+  packages:
+    - binutils
+    - scanelf
+
+inputs:
+  opts:
+    description: |
+      The option flags to pass to the strip command.
+    default: -g
+
+pipeline:
+  - working-directory: ${{targets.contextdir}}
+    runs: |
+      scanelf --recursive --nobanner --etype "ET_DYN,ET_EXEC" \
```

Comment:
shell sucks.
left side of this pipe fails silently and results in nothing being done.

Reply:
I'd love for this to get put into [chainguard-dev/tw](https://github.com/chainguard-dev/tw)